xml/abstract_commands.xml

```diff
@@ -55,6 +57,9 @@
         This command modifies {\tt arg0} only when a register is read. The
         other {\tt data} registers are not changed.
 
+	For the Access Register Command, the \RdmCommand register (see Section~\ref{command})
```

Comment:
This seems unnecessary. We don't have a statement like that for any other register/command definition. Why do we need one here?

Reply:
This provides a reference to the command register whose content is being further specified.  While this may be obvious to those familiar with the spec, it is not for new readers.  In my first readings of the spec, I missed seeing appropriate links and it takes extra reading to be confident to have correctly connected all the dots.  You're right that a similar statement is needed for other commands, to be consistent.

Reply:
If we want this, let's do it right and do it everywhere. Can you remove this line from this PR? I've opened #471 to track the broader issue. (Please feel free to remind me about that one if I don't follow up.)